.cirrus.yml

```diff
@@ -16,17 +22,13 @@ test_task:
     - sysctl vfs.aio.enable_unsafe=1
   cargo_cache:
     folder: $HOME/.cargo/registry
-    fingerprint_script: cat Cargo.lock || echo ""
-  nightly_build_script:
+    fingerprint_script:
+      - echo $VERSION
+      - cat Cargo.lock || echo ""
+  build_script:
     - . $HOME/.cargo/env
-    - cargo +nightly-2019-09-26-x86_64-unknown-freebsd build --all-features --all
-  stable_build_script:
+    - cargo $VERSION build --all-features --all
```

Comment:
This step is going to fail with the stable compiler, because `--all-features` requires nightly.  That's why I originally used separate tasks for stable vs nightly rather than use `matrix`.